broker/applications/osb-broker/src/api-controllers/ServiceBrokerApiController.js

```diff
@@ -202,7 +202,10 @@ class ServiceBrokerApiController extends FabrikBaseController {
       res.status(statusCode).send(body);
     }
 
-    const labels = req.params.region ? { 'region':req.params.region } : undefined;
+    let labels = req.params.region ? { 'region':req.params.region } : undefined;
+    labels = _.merge(labels, {
```

Comment:
Here were are updating the label whenever we gets a plan update request. But I feel there can be a case when the user sends a wrong plan to update to and our `isUpdatePossible` assertion rejects it. Then we may have a wrong plan id in the label.

Reply:
Sorry, I realized that we are not applying the label is there are errors. 👍